bundles/org.openhab.binding.ipobserver/README.md

```diff
@@ -15,15 +19,20 @@ There is only one thing that can be added and is called `weatherstation`.
 
 ## Discovery
 
-Auto discovery is supported and may take a while to complete as it scans all IP addresses on your network one by one.
+Auto discovery is supported for the RJ45 models, and the wifi ipobservers will need to be manually added. Discovery may take a while to complete as it scans all IP addresses on your network one by one.
```

Comment:
Please go to a new line after each `.`
I am certainly not fluent in English but isn't "while" more appropriate in this case ? Auto discovery is supported for the RJ45 models, **while** the wifi ipobservers will need to be manually added.

Reply:
Yes `while` sounds much better, thank you.